db/schema.rb

```diff
@@ -69,12 +69,6 @@
     t.index ["rspec_report_id"], name: "index_rspec_summaries_on_rspec_report_id"
   end
 
-  create_table "scenarios", force: :cascade do |t|
-    t.bigint "project_id"
-    t.string "description"
-    t.index ["project_id"], name: "index_scenarios_on_project_id"
-  end
-
```

Comment:
Is this not being used?

Reply:
I ran the migration and that was the result for me 🤔